packages/cms-lib/package.json

```diff
@@ -19,6 +19,7 @@
     "ignore": "^5.1.4",
     "js-yaml": "^3.12.2",
     "moment": "^2.24.0",
+    "open": "6.4.0",
```

Comment:
Is this needed? Looks like the `open` require is in cms-cli which has the dep already.

Reply:
`open` is used in both `cms-cli` and `cms-lib` now.